test/integration/buildclient_test.go

```diff
@@ -219,7 +219,16 @@ func NewTestBuildOpenshift(t *testing.T) *testBuildOpenshift {
 		Stop: openshift.stop,
 	}
 
-	factory.Create().Run()
+	bcFactory.Create().Run()
+
+	bpcFactory := buildcontrollerfactory.BuildPodControllerFactory{
```

Comment:
In the future I might just have had one factory have two "Create" methods.  I think this is ok for now until we stop adding / changing controllers, and then we can look to simplify some of this logic.